packages/ckeditor5-dev-changelog/src/utils/getnewversion.ts

```diff
@@ -43,6 +52,13 @@ export async function getNewVersion(
 		bumpType = 'major';
 	}
 
+	if ( noWrite ) {
+		return {
+			newVersion: semver.inc( oldVersion, bumpType ) || oldVersion,
+			isInternal: false
+		};
+	}
+
```

Comment:
Why did you add these? The issue you work on does not say anything about updating versions. We aim to resolve a promise and prevent the removal of input files. Am I missing anything, or are the changes out of the scope?

Reply:
Since we want to run those options in an automatic script, we need to prevent `inquirer` from running, since it would block the script.

Reply:
For this purpose we have the `nextVersion` option.